Sources/Sentry/SentryAutoSpanOnScopeStarter.m

```diff
@@ -0,0 +1,17 @@
+#import "SentryAutoSpanOnScopeStarter.h"
+#import "SentryHub+Private.h"
+#import "SentrySDK+Private.h"
+#import "SentryScope+Private.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation SentryAutoSpanOnScopeStarter
+
+- (void)startSpan:(SentrySpanCallback)callback
```

Comment:
m: Im not sure about this name.
The function actually don't start any span, it gives you the span in scope to use it as necessary, and not necessarily to start another span.